content/python/concepts/built-in-functions/terms/tuple/tuple.md

```diff
@@ -0,0 +1,34 @@
+---
+Title: 'tuple()'
+Description: 'Creates a new tuple.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Collections'
+  - 'Tuples'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The built-in **`tuple()`** function creates a new [tuple](https://www.codecademy.com/resources/docs/python/tuples).
+
+## Syntax
+
+```pseudo
+tuple(iterable)
```

Comment:
I noticed in other documentation that we can also create a tuple like so:

`my_tuple = (value1, value2, ...)`

Values are optional here. If no values are added, an empty tuple is created.

Did we also want to include that data types can be mixed?

Reply:
I also saw something about unpacking a tuple which looked cool~